modules/AWS/eks/osquery.tf

```diff
@@ -0,0 +1,45 @@
+// Policy created following this guide: http://go/osquery-setup
+// Microscope has to be manually created at https://microscope.prod.atl-paas.net/services/${var.cluster_name}
+
+locals {
+    account_id = data.aws_caller_identity.current.account_id
+}
+
+# See: https://hello.atlassian.net/wiki/spaces/OBSERVABILITY/pages/140624694/Logging+pipeline+-+Sending+logs+to+Splunk#Kinesis-Stream-Details
+data "aws_iam_policy_document" "laas" {
+  count       = var.osquery_secret_name != "" ? 1 : 0
+  statement {
+    effect    = "Allow"
+    actions   = ["sts:AssumeRole"]
+    resources = ["arn:aws:iam::915926889391:role/pipeline-prod-log-producers-all",
```

Comment:
Shall we pass it in as variable? so that aws account id won't be publicly visible.

Reply:
Yes, it does make sense. Added arns as a variable, also added to secrets in GH actions